test/Formik.test.tsx

```diff
@@ -9,6 +9,7 @@ jest.spyOn(global.console, 'warn');
 
 interface Values {
   name: string;
+  zipcode?: number;
```

Comment:
formatting: `zipCode`, although since international users use different terminology / formats maybe `age` would be better.

Reply:
Updated to use age instead of zip code